### PR TITLE
ZIndex Classes: Docs fixes

### PR DESCRIPTION
**Pull request by AlbertCarreras**
### Links

- [Jira](link to Jira ticket(s))
- [TDD](link to Paper doc)
- [Figma](link to Figma file)

### Checklist

- [ ] Added unit and Flow Tests
- [ ] Added documentation + accessibility tests
- [ ] Verified accessibility: keyboard & screen reader interaction
- [ ] Checked dark mode, responsiveness, and right-to-left support
- [ ] Checked stakeholder feedback (e.g. Gestalt designers)

### CODE REVIEW

File: docs/src/ZIndex Classes.doc.js [hunk 1]
```diff
@@ -140,7 +148,7 @@ const BoxWithZIndex = <Box zIndex={composedZIndex}/>
         description={`
 Use the lowest possible z-index values and compose them.
 ~~~jsx
-import { FixedZIndex } from 'gestalt';
+import { ComposedZIndex, FixedZIndex } from 'gestalt';
```
Comment (dangerismycat) [inline docs/src/ZIndex Classes.doc.js:151]: `CompositeZIndex`?

File: docs/src/ZIndex Classes.doc.js [hunk 2]
```diff
@@ -67,10 +67,12 @@ card(
 Export FixedZIndex and CompositeZIndex rather than the z-index values itself.
 ~~~jsx
 // exporting file
+import { FixedZIndex } from 'gestalt';
+
 export const BaseZIndex = = new FixedZIndex(1);
 
 // importing file
-import { BaseZIndex } from 'gestalt';
+import { BaseZIndex } from 'file';
```
Comment (dangerismycat) [inline docs/src/ZIndex Classes.doc.js:75]: This makes it seem like the import is from [this npm package](https://www.npmjs.com/package/File)…consider something like `'./path/to/your/zindex/file.js'`

File: docs/src/ZIndex Classes.doc.js (hunk 1 above)
Comment (AlbertCarreras) [reply docs/src/ZIndex Classes.doc.js]: DONE

File: docs/src/ZIndex Classes.doc.js (hunk 2 above)
Comment (AlbertCarreras) [reply docs/src/ZIndex Classes.doc.js:75]: DONE

Comment (dangerismycat) [reply docs/src/ZIndex Classes.doc.js:75]: I don't think you committed that change? Or you didn't push it up?

Comment (AlbertCarreras) [reply docs/src/ZIndex Classes.doc.js:75]: mmm i did, maybe didn't push.

Comment (AlbertCarreras) [reply docs/src/ZIndex Classes.doc.js:75]: maybe i didn't///

**Review by dangerismycat — APPROVED** [review]

**Pull request merged**